pymongo/encryption_options.py

```diff
@@ -45,9 +45,10 @@ def __init__(
         mongocryptd_spawn_path: str = "mongocryptd",
         mongocryptd_spawn_args: Optional[List[str]] = None,
         kms_tls_options: Optional[Mapping[str, Any]] = None,
+        encrypted_fields_map: Optional[Mapping] = None,
+        bypass_query_analysis: bool = False,
         crypt_shared_lib_path: Optional[str] = None,
         crypt_shared_lib_required: bool = False,
-        bypass_query_analysis: bool = False,
```

Comment:
The order should be:
```
        crypt_shared_lib_required: bool = False,
        bypass_query_analysis: bool = False,
        encrypted_fields_map: Optional[Mapping] = None,
```

When adding new args, we always append to the end to maintain backwards compat for positional args.